qiita_db/data.py

```diff
@@ -416,6 +416,17 @@ def _remove_filepath(self, fp, conn_handler, queue):
             self._set_link_filepaths_status("failed: %s" % msg)
             raise QiitaDBError(msg)
 
+        # The filepath belongs to one or more studies
+        studies_linked = self.studies
```

Comment:
@josenavas and I just discussed offline and I will remove the
possibility of linking more files via the GUI to a raw data if it has
been shared. One of my next PRs will be to erase a raw data from a
study. This will allow that if a raw data is shared between n studies,
n-1 studies could remove the raw data, and the last one can
link/unlink files.